src/theme.cpp

```diff
@@ -144,3 +148,10 @@ QVariant Theme::data(const QModelIndex& index, int role) const {
       return QVariant();
   }
 }
+
+void Theme::setStatusBarTextColor(
+    [[maybe_unused]] const StatusBarTextColor color) {
```

Comment:
Nice! We often use a macro for this:
`Q_UNUSED(color);`

The code would be:
```
#ifdef MVPN_IOS
  ...
#else
  Q_UNUSED(color)
#endif
```

But I like your solution too!